apps/stats/models.py

```diff
@@ -88,17 +88,17 @@ class Meta:
 
 class ThemeUpdateCountManager(models.Manager):
 
-    def get_range_days_avg(self, start, end):
-        """Return a a dict of the average number of users (count) over the
-        given range of days, per addons."""
-        averages = (self.values('addon_id')
-                        .filter(date__range=[start, end])
-                        .annotate(avg=models.Avg('count')))
-        # Transform the queryset from a list of dicts
-        #   [{'addon_id': id1, 'count__avg': avg1], {'addon_id': id2, ...
-        # to a dict
-        #   {id1: avg1, id2: avg2, ...}
-        return dict((d['addon_id'], d['avg']) for d in averages)
+    def get_range_days_avg(self, start, end, *extra_fields):
```

Comment:
I wanted to do that but I needed to return 2 elements tuples for the `prev_3_weeks_avgs = dict(` call that the command makes. I'm open to better suggestions...